config/config.exs

```diff
@@ -14,6 +14,37 @@ config :search,
 # Add types added by the pgvector-elixir extension to Postgrex
 config :search, Search.Repo, types: Search.PostgrexTypes
 
+# Register embedding providers
+config :search, :embedding_providers,
+  paraphrase_l3: {
+    Search.Embeddings.BumblebeeProvider,
+    serving_name: Search.Embeddings.ParaphraseL3,
+    table_name: "paraphrase_l3_embeddings",
```

Comment:
Since we already pike a name for the model key, I would generate the table name based on it, with a common prefix as in in `"embeddings__#{model_key}"`.